### PR TITLE
feat(browser): LP to PP release of browser SPA

### PR DESCRIPTION
**Pull request by ghost**
This PR covers moving the LP browser doc "Browser SPA 2.0" from beta repo to public repo. The only modifications needed for PP is to update the banner at the top of the page.

### CODE REVIEW

File: src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    </Step>
+
+    <Step>
+        ### Add the feature flag [#add-feature-flag]
+
+        If your agent was installed with the APM method, contact our support team and we'll enable the feature flag on your account.
+        
+        If your agent was installed with NPM or the UI-based, copy/paste method, add the following feature flag your browser agent code:
+
+            1. Find the New Relic browser agent code in your webpage HTMl.
```
Comment (cbehera-newrelic) [inline src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx:49]: HTML webpage. (L should be in uppercase)

File: src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+        Next, check the network tab in your browser’s dev tools. Filter for requests going to `/events/1/`. There should be ones for which the payload begins with `bel.7;1`, indicating your interactions are being captured and sent.
+
+        Finally, interaction data should continue to flow into your browser entity dashboards and charts.
+
+    </Step>
+
+</Steps>
+
+## Known issues [#known-issues]
```
Comment (cbehera-newrelic) [inline src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx:76]: Should it be a part of release note? Or is it okay to place it here?

**Review by cbehera-newrelic — COMMENTED** [review]
I have left minor comments on it and have few doubts. Looks like release statements.

File: src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx (hunk 1 above)
Comment (ally-sassman) [inline src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx:49]:
```suggestion
            1. Find the New Relic browser agent code in your webpage HTML.
```

File: src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx (hunk 2 above)
Comment (ally-sassman) [reply src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx:76]: It's okay to document known issues only for a limited/public preview where we're still working out the kinks.

**Pull request merged**